packages/accounts-password/password_server.js

```diff
@@ -582,9 +535,14 @@ Accounts.sendResetPasswordEmail = function (userId, email) {
  * @param {String} [email] Optional. Which address of the user's to send the email to. This address must be in the user's `emails` list. Defaults to the first email in the list.
  */
 Accounts.sendEnrollmentEmail = function (userId, email) {
-  // XXX refactor! This is basically identical to sendResetPasswordEmail.
+  Accounts.sendEmail(userId, email, 'enrollAccount');
+};
 
-  // Make sure the user exists, and email is in their addresses.
+/**
+ * @param {String} type Can be either resetPassword or enrollAccount depending on which function called it.
```

Comment:
What about email? Shouldn't that also be in the params list, as it's present in the other send...Email functions